src/System.Drawing.Primitives/src/System/Drawing/SizeF.cs

```diff
@@ -184,6 +209,16 @@ public float Height
         ///    </para>
         /// </summary>
         public override string ToString() => "{Width=" + _width.ToString() + ", Height=" + _height.ToString() + "}";
```

Comment:
Minor nit - can use string interpolation instead of adding strings with + and ToString()